src/components/Login.js

```diff
@@ -4,6 +4,7 @@ import { library } from "@fortawesome/fontawesome-svg-core";
 import { fab } from "@fortawesome/free-brands-svg-icons";
 import FlashMessage from "react-flash-message";
 import "./styles/login.css";
+import "./styles/privacy.css";
```

Comment:
you could put the styling inside login css instead of having to create a separate css file. just add `.privacy-link > a` into `login.css`